.eslintrc.js

```diff
@@ -11,6 +11,7 @@ module.exports = {
     'plugin:prettier/recommended',
     'airbnb-base',
     'airbnb-typescript/base',
+    "prettier"
```

Comment:
Why do you add this? We have alredy `plugin:prettier/recommende`. It is work well.

Reply:
to resolve ESLint rules and Prettier conflict. with only 'plugin:prettier/recommended', it does't work.